fs2/src/main/scala/uk/gov/nationalarchives/dp/client/fs2/Fs2Client.scala

```diff
@@ -3,14 +3,40 @@ package uk.gov.nationalarchives.dp.client.fs2
 import cats.effect.IO
 import sttp.capabilities.fs2.Fs2Streams
 import sttp.client3.httpclient.fs2.HttpClientFs2Backend
-import uk.gov.nationalarchives.dp.client.Client._
-import uk.gov.nationalarchives.dp.client.Client
-
+import uk.gov.nationalarchives.dp.client.EntityClient._
+import uk.gov.nationalarchives.dp.client.AdminClient._
+import uk.gov.nationalarchives.dp.client.ContentClient.createContentClient
+import uk.gov.nationalarchives.dp.client.{AdminClient, ContentClient, EntityClient}
+import uk.gov.nationalarchives.dp.client.Utils.ClientConfig
 import scala.concurrent.duration._
 
 object Fs2Client {
-  def client(url: String, duration: FiniteDuration = 15.minutes): IO[Client[IO, Fs2Streams[IO]]] =
+  private val defaultSecretsManagerEndpoint = "https://secretsmanager.eu-west-2.amazonaws.com"
```

Comment:
Mostly the reason this is here is because it makes testing easier and it's quite fiddly to override environment variables in tests. Each test has it's own port on Wiremock to avoid clashes when they're run in parallel as well so I can't just add the environment variable to the build.sbt. 
I think we can leave it. There is a possibility that we might need to change the endpoint eventually if we get some odd VPC setup but I think it's ok for now.